app/build.gradle

```diff
@@ -9,7 +9,7 @@ android {
     buildToolsVersion "29.0.0"
     defaultConfig {
         applicationId "com.example.una"
-        minSdkVersion 16
+        minSdkVersion 24
```

Comment:
you don't want your minSdk high. You will eliminate almost 40% of your user basehttps://developer.android.com/about/dashboards